app/overrides/decidim/proposals/proposals/edit/remove_title.html.erb.deface

```diff
@@ -0,0 +1 @@
+<!-- remove ".hashtags__container" -->
```

Comment:
Do we need this deface entry? I don't see any `.hashtags__container" in the `edit.html.erb`